server/openapi_server/db_connection.py

```diff
@@ -1,41 +1,40 @@
-import psycopg2
-import psycopg2.extras
 import logging
-from pathlib import Path
-from configparser import ConfigParser
-import logging
-import os.path
 import os
+import os.path
 import xml.etree.ElementTree as ET
 
+import psycopg2
+import psycopg2.extras
+
+from openapi_server.util.configuration import CD2HConfig as config
 
-def load_config(config_file="database.ini"):
+
+def load_config():
     """
-    Load the configuration file
-    :param config_file:
-    :return:
+    Load the configuration file needed by the database from the config class
+    :return: None
     """
     db = {}
-    db['user'] = os.environ["SQL_USER"]
-    db['password'] = os.environ["SQL_PASSWORD"]
-    db['host'] = os.environ["SQL_HOST"]
-    db['database'] = os.environ["SQL_DB"]
-    db['port'] = os.environ["SQL_PORT"]
+    db['user'] = config().sql_user
+    db['password'] = config().sql_password
+    db['host'] = config().sql_host
+    db['database'] = config().sql_db
+    db['port'] = config().sql_port
 
     return db
 
 
 def get_connection_local_pg(params):
     """
-    Return the Postgres db Connection
+    Return a Postgres db Connection
```

Comment:
Because the code should work with any SQL implementation (I think), let's have the doc referring to SQL instead of a specific implementation.